frontend/www/js/omegaup/components/arena/Scoreboard.vue

```diff
@@ -165,6 +165,7 @@ export default class ArenaScoreboard extends Vue {
   @Prop() problems!: omegaup.Problem[];
   @Prop() ranking!: types.ScoreboardRankingEntry[];
   @Prop() lastUpdated!: Date;
+  @Prop() shouldShowInvitedUsersFilter!: boolean;
```

Comment:
```suggestion
  @Prop({ default: true }) showInvitedUsersFilter!: boolean;
```

por consistencia con la línea siguiente.

Reply:
Listo